### DISCUSSION TITLE
What keeps you going in the trenches? A thread for inspiration, productivity tips, relaxation techniques, etc.

### DISCUSSION THREAD

**Discussion by radtkea** [IBEXImagingCommunity/ibex_imaging_knowledge_base#71] [Q&A]
Science is tough. It's easy to feel overwhelmed, out of time, alone, discouraged, and burned out. 

Here's one thing I do that helps: The [Pomodoro Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique). My mother and I do it on Zoom together. She's retired and works on many projects around the house and garden. We work together:)

**Comment by nzangger** [comment]
Big fan of the Pomodoro myself with some music in the background! It gets me through late evenings :) 

https://www.youtube.com/watch?v=grBFMP3HDZA&t=937s&ab_channel=AbaoinTokyo

**Reply by radtkea** [reply]
This is amazing! I love this. Thank you very much for sharing.

**Reply by anita-gola** [reply]
Dear Nathan- I also love the Pomodoro technique (kindly shared by AJR here ^).. and really loving the jam you posted! Thanks for posting!

**Reply by btwachter** [reply]
Dear Nathan, 

Andrea mentioned you introduced her to "study with me" videos and asked me to share my favorite. I feel like this YouTuber got me through the last few semesters of college (especially when classes were virtual during the pandemic and I could not study with friends). https://youtu.be/wrnM-6owLLs

And a holiday themed version that was lovely when finals were schedule right before Christmas. :) https://youtu.be/DK_NnMvV87s

Hope you had safe travels home and a very productive internship!

Brianna
